File: RepoTransBench/target_projects/Java/C#/lhazlewood_apache-shiro-tutorial-webapp/public_tests/MetaPublicTest.cs
using System;
using Xunit;
using System.IO;

namespace PublicTests
{
    public class MetaPublicTest
    {
        [Fact]
        public void TestProjectRequiredFilesExist_Public()
        {
            Assert.True(File.Exists("pom.xml"), "pom.xml must be present in the repo");
            Assert.True(File.Exists("LICENSE"), "LICENSE should exist in project root");
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/lhazlewood_apache-shiro-tutorial-webapp/public_tests/meta_public_test.go
package public_tests

import (
	"os"
	"testing"
)

func TestMetaPublicTest_ProjectRequiredFilesExist_Public(t *testing.T) {
	if _, err := os.Stat("pom.xml"); err != nil {
		t.Errorf("pom.xml must be present in the repo, err: %v", err)
	}
	if _, err := os.Stat("LICENSE"); err != nil {
		t.Errorf("LICENSE should exist in project root, err: %v", err)
	}
}